src/main/webapp/js/webrtc_adaptor.js

```diff
@@ -583,15 +595,15 @@ export class WebRTCAdaptor
 			composedStream.addTrack(videoTrack);
 		});
 
-		this.audioContext = new AudioContext();
-		var audioDestionation = this.audioContext.createMediaStreamDestination();
+		var audioContext = new AudioContext();
```

Comment:
Why have we changed this context from field to local variable? 
As far as I remember, there was a refactor commit previously that made the reverse.